scripts/process/clean_up_raw.R

```diff
@@ -0,0 +1,12 @@
+process.clean_up_raw <- function(viz = as.viz("clean_up_raw")){
+  library(dplyr)
+  library(data.table)
+  
+  viz.data <- readDepends(viz)[["fetchGA_cida"]]
+  
+  viz.data <- viz.data %>%
+    mutate(date = as.Date(paste(year,month,day,sep="-"), origin = "1970-01-01")) %>%
```

Comment:
I'm going to leave this in until your PR comes in actually (otherwise, my stuff here will break)